test/built-ins/BigInt/asIntN/arithmetic.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
```

Comment:
esid should be pending when an id is not available. In this case, we can have a prediction in the proposal. `esid: pending` should be the last resource, as it's becoming common to see tests left this way without any fix by the original contributors. The sooner we have it, better. Otherwise, this will cause an extra maintenance cost.